sdk/keyvault/Azure.Security.KeyVault.Keys/README.md

```diff
@@ -115,113 +115,115 @@ The following section provides several code snippets using the [above created](#
 
-```C# DeleteKey
-DeletedKey key = client.DeleteKey("key-name");
-
-Console.WriteLine(key.Name);
-Console.WriteLine(key.DeletedDate);
+```C# DeleteKey
+DeletedKey key = client.DeleteKey("key-name");
+
+Console.WriteLine(key.Name);
+Console.WriteLine(key.DeletedDate);
 ```
 
 ### List Keys
 This example lists all the keys in the specified Key Vault.
 
-```C# ListKeys
-Pageable<KeyProperties> allKeys = client.GetKeys();
-
-foreach (KeyProperties key in allKeys)
-{
-    Console.WriteLine(key.Name);
-}
+```C# ListKeys
+Pageable<KeyProperties> allKeys = client.GetKeys();
+
+foreach (KeyProperties key in allKeys)
+{
+    Console.WriteLine(key.Name);
+}
 ```
 
 ### Encrypt and Decrypt
 This example creates a CryptographyClient and uses it to encrypt and decrypt with a key in Key Vault.
 
-```C# EncryptDecrypt
-byte[] plaintext = Encoding.UTF8.GetBytes("A single block of plaintext");
-
-// encrypt the data using the algorithm RSAOAEP
-EncryptResult encryptResult = cryptoClient.Encrypt(EncryptionAlgorithm.RsaOaep, plaintext);
-
-// decrypt the encrypted data.
-DecryptResult decryptResult = cryptoClient.Decrypt(EncryptionAlgorithm.RsaOaep, encryptResult.Ciphertext);
+```C# EncryptDecrypt
+byte[] plaintext = Encoding.UTF8.GetBytes("A single block of plaintext");
+
+// encrypt the data using the algorithm RSAOAEP
+EncryptResult encryptResult = cryptoClient.Encrypt(EncryptionAlgorithm.RsaOaep, plaintext);
+
+// decrypt the encrypted data.
+DecryptResult decryptResult = cryptoClient.Decrypt(EncryptionAlgorithm.RsaOaep, encryptResult.Ciphertext);
 ```
 
 ### Async create a Key
 Async APIs are identical to their synchronous counterparts. Note that all methods end with `Async`.
 
-```C# CreateKeyAsync
-// Create a key of any type
-Key key = await client.CreateKeyAsync("key-name", KeyType.Rsa);
-
-Console.WriteLine(key.Name);
-Console.WriteLine(key.KeyMaterial.KeyType);
-
-// Create a software RSA key
-var rsaCreateKey = new RsaKeyCreateOptions("rsa-key-name", hsm: false);
-Key rsaKey = await client.CreateRsaKeyAsync(rsaCreateKey);
-
-Console.WriteLine(rsaKey.Name);
-Console.WriteLine(rsaKey.KeyMaterial.KeyType);
-
-// Create a hardware Elliptic Curve key
-var echsmkey = new EcKeyCreateOptions("ec-key-name", hsm: true);
-Key ecKey = await client.CreateEcKeyAsync(echsmkey);
-
-Console.WriteLine(ecKey.Name);
-Console.WriteLine(ecKey.KeyMaterial.KeyType);
+```C# CreateKeyAsync
+// Create a key of any type
+Key key = await client.CreateKeyAsync("key-name", KeyType.Rsa);
+
+Console.WriteLine(key.Name);
+Console.WriteLine(key.KeyMaterial.KeyType);
+
+// Create a software RSA key
+var rsaCreateKey = new RsaKeyCreateOptions("rsa-key-name", hsm: false);
+Key rsaKey = await client.CreateRsaKeyAsync(rsaCreateKey);
+
+Console.WriteLine(rsaKey.Name);
+Console.WriteLine(rsaKey.KeyMaterial.KeyType);
+
+// Create a hardware Elliptic Curve key
```

Comment:
Probably should include the same note about needing a premium SKU here as you did above.